integration-cli/trust_server.go

```diff
@@ -106,7 +106,7 @@ func newTestNotary(c *check.C) (*testNotary, error) {
 	}
 
 	// Wait for notary to be ready to serve requests.
-	for i := 1; i <= 5; i++ {
+	for i := 1; i <= 20; i++ {
```

Comment:
@aaronlehmann let's give this a try; we can raise it further if it's still flaky. If it is, we may want to temporarily add some logging to see how long it actually takes, sgty?